net-wireless/wpa_supplicant/wpa_supplicant-2.6-r8.ebuild

```diff
@@ -12,13 +12,13 @@ LICENSE="|| ( GPL-2 BSD )"
 
 SLOT="0"
 KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd"
-IUSE="ap bindist dbus eap-sim eapol_test fasteap gnutls +hs2-0 libressl p2p privsep ps3 qt5 readline selinux smartcard ssl suiteb tdls uncommon-eap-types wimax wps kernel_linux kernel_FreeBSD"
+IUSE="ap bindist +crda dbus eap-sim eapol_test fasteap gnutls +hs2-0 libressl p2p privsep ps3 qt5 readline selinux smartcard ssl suiteb tdls uncommon-eap-types wimax wps kernel_linux kernel_FreeBSD"
```

Comment:
it requires revbump.

Reply:
Thanks for your feedback. I thought to remember that revbumps are not required in case changes only impact the USE flags. I just took a look at my mail archives and found a [message](https://archives.gentoo.org/gentoo-dev/message/8a2d6f2d4b153e4ae399cfbd3dc25ffd) from William Hubbs which states that, but it's from 2015. I'm happy to do the revdump though in case it is necessary